### PR TITLE
[reCaptcha v3]: Hide reCAPTCHA badge on phone and show our own…

### PR DESCRIPTION
**Pull request by p-jackson**
Trying out just leaving the badge where it is on desktop. The signup is already very wordy and at least the badge is tucked out of the way. However on small screens the floating badge gets in the way so that's when we show our own message.

[The specific wording comes from Google.](https://developers.google.com/recaptcha/docs/faq#id-like-to-hide-the-recaptcha-badge.-what-is-allowed)

Hiding the badge at 660px wide because that's when the login buttons jump to the bigger size.

Epic: Automattic/zelda-private#239

##### Desktop

Passwordless | With Password
-------------|-----
passwordless desktop | password desktop

##### Phone

Passwordless | With Password
-------------|-----
passwordless phone | password phone

#### Testing instructions

* Follow setup steps in #37170
* Resize browser to observe.

### CODE REVIEW

**Comment by p-jackson** [comment]
What do you think of the idea of just showing the badge on desktop @jancavan? Not a fan of adding non-a8c branding to the page, but at least it keeps it out of the way.

My only other idea was a Sticky Footer, but they can be notoriously tricky to get right.

**Comment by adamziel** [comment]
Can we hide the badge entirely?

**Comment by p-jackson** [comment]
@adamziel yup we can remove the badge as long as we have the `This site is protected by reCAPTCHA and the Google Privacy Policy and Terms of Service apply.` blurb somewhere. I just couldn't find a good place to put the text that didn't look gross.

Added [Design Needed].

**Comment by adamziel** [comment]
Let's also explicitly ping @jancavan

**Comment by jancavan** [comment]
@p-jackson @adamziel I'll take a look at this today.

**Comment by jancavan** [comment]
The desktop version with the badge looks fine.

I'm wondering if we can just omit the first "Terms of Service" link so it doesn't have to be there twice:

recaptcha001

Even with one less line, the text with the blue background is still a bit wordy. Retaining the existing elements, I think what we could do is to just make sure we increase/have equal spacing in between sentences. I also used a different color for the Google line.

**Before:**
Screenshot 2019-11-05 10 18 35


**After:**
recaptcha002

@p-jackson @adamziel

**Comment by p-jackson** [comment]
I've increased the space around the text elements on mobile, I've left the desktop layout as it is. Also reduced the contrast Google ToS text, I'm not sure of the exact colour you used @jancavan but I've gone right on the borderline of what's acceptable according to the WCAG guidelines, I wouldn't want to go any lower.

Nov-06-2019 09-41-36

Static image to inspect padding and colour more closely:
Screenshot 2019-11-06 at 9 54 07 AM

Looks like there's already a discussion underway w.r.t the doubling up of ToS links paObgF-E4-p2

**Comment by p-jackson** [comment]
Actually now that I think of it, that recaptcha badge on desktop is going to interfere with the help button. Hopefully we're able to find a way to remove the badge once we're done with it @andrewserong

**Comment by jancavan** [comment]
> I'm not sure of the exact colour you used

I'm not sure what the corresponding color is we use in code, but it's `#c5d9ed` from our https://color-studio.blog/

Spacing looks good to me.

> Actually now that I think of it, that recaptcha badge on desktop is going to interfere with the help button.

I didn't realize the floating Help button is actually shown in Signup/onboarding?

**Comment by jancavan** [comment]
> Looks like there's already a discussion underway w.r.t the doubling up of ToS links paObgF-E4-p2

@p-jackson Thanks for pointing me to that link. It looks like once the PR for it is merged: https://github.com/Automattic/wp-calypso/pull/36295, we'll have a less wordy page.

**Comment by p-jackson** [comment]
Thanks for the colour code @jancavan :)

> I didn't realize the floating Help button is actually shown in Signup/onboarding?
It's only shown when logged in. So if you create a new site from calypso, or if you refresh the page after after signing up.

**Comment by andrewserong** [comment]
This is looking good @p-jackson — just needs a rebase, then I think we can merge it into the other PR. Or should we wait until the string freeze on this one is complete?

**Comment by p-jackson** [comment]
Thanks @andrewserong, rebased 👍 
Ya we could merge now (we can just add [String Freeze] to the parent PR. But I was thinking of merging this _after_ #37170 was approved. That way each independent part can be reviewed individually and we don't have any uber PRs that need reviewed.
It means updating a PR after the review is complete, which might technically be naughty, but we'll only be updating it with other approved code 🤷‍♂

**Comment by andrewserong** [comment]
Actually, that’s a great idea @p-jackson — let’s wait and avoid the uber PR 😊

**Comment by p-jackson** [comment]
Need to fix the phone view for passwordless signup. It currently shows the badge even in the phone layout, which overlaps user controls.

**Comment by p-jackson** [comment]
Fixed up an earlier bad rebase and added abtest checks to the UI changes.
We can remove [String Freeze] as this is now in an English only abtest.

**Comment by andrewserong** [comment]
Looks great @p-jackson and tests nicely across Chrome, FF, Safari, and IE11. The only tiny thing is our naming of reCAPTCHA, here recaptcha is written in camel case `showReCaptchaToS` but elsewhere in the other PR we're using e.g. `initGoogleRecaptcha`. Do you think it's worth changing this one (or the references in the other PR)?

**Comment by p-jackson** [comment]
Thanks @andrewserong I was just thinking of that. Seems like everywhere else we've been treating it as one word, so I've renamed it to `showRecaptchaToS`.

**Comment by andrewserong** [comment]
The failing e2es appear unrelated, and manually stepping through the e2e scenario (selecting all sites and activating a theme) works for me from this calypso.live branch.

**Comment by p-jackson** [comment]
Re-ran tests and they passed. Merging into uber PR now..
